WordPress/Sniffs/WP/DeprecatedParametersSniff.php

```diff
@@ -0,0 +1,352 @@
+			),
+		),
+		'trackback_url' => array(
+			1 => array(
+				'value'   => true,
+				'version' => '2.5.0',
+			),
+		),
+		'update_user_status' => array(
+			4 => array(
+				'value'   => null,
+				'version' => '3.0.2',
+			),
+		),
+		'update_blog_status' => array(
+			4 => array(
+				'value'   => null,
+				'version' => '3.1.0',
+			),
+		),
+		'update_blog_option' => array(
+			4 => array(
+				'value'   => null,
+				'version' => '3.1.0',
+			),
+		),
+		'unregister_setting' => array(
+			4 => array(
+				'value'   => '',
+				'version' => '4.7.0',
+			),
+		),
+		'wp_get_http_headers' => array(
+			2 => array(
+				'value'   => false,
+				'version' => '2.7',
+			),
+		),
+		'wp_get_sidebars_widgets' => array(
+			1 => array(
+				'value'   => true,
+				'version' => '2.8.1',
+			),
+		),
+		'wp_install' => array(
+			5 => array(
+				'value'   => '',
+				'version' => '2.6.0',
+			),
+		),
+		'wp_new_user_notification' => array(
+			2 => array(
+				'value'   => null,
+				'version' => '4.3.1',
+			),
+		),
+		'wp_notify_postauthor' => array(
+			2 => array(
+				'value'   => null,
+				'version' => '3.8.0',
+			),
+		),
+		'wp_title_rss' => array(
+			1 => array(
+				'value'   => '&#8211;',
+				'version' => '4.4.0',
+			),
+		),
+		'wp_upload_bits' => array(
+			2 => array(
+				'value'   => '',
+				'version' => '2.0.0',
+			),
+		),
+		'xfn_check' => array(
+			3 => array(
+				'value'   => '',
+				'version' => '2.5.0',
+			),
+		),
+
+	); // End $target_functions.
+
+	/**
+	 * Process the parameters of a matched function.
+	 *
+	 * @since 0.11.0
+	 *
+	 * @param int    $stackPtr        The position of the current token in the stack.
+	 * @param array  $group_name      The name of the group which was matched.
+	 * @param string $matched_content The token content (function name) which was matched.
+	 * @param array  $parameters      Array with information about the parameters.
+	 *
+	 * @return void
+	 */
+	public function process_parameters( $stackPtr, $group_name, $matched_content, $parameters ) {
+		$paramCount = count( $parameters );
+		foreach ( $this->target_functions[ $matched_content ] as $position => $parameter_args ) {
+
+			if ( $position > $paramCount ) {
+				break;
+			}
+
+			if ( ! isset( $parameters[ $position ] ) ) {
+				continue;
+			}
+
+			switch ( $parameters[ $position ]['raw'] ) {
+				case 'true':
+					$matched_parameter = true;
+					break;
+				case 'false':
+					$matched_parameter = false;
+					break;
+				case 'null':
+					$matched_parameter = null;
+					break;
+				case 'array()':
+					$matched_parameter = array();
+					break;
+				default:
+					$matched_parameter = $this->strip_quotes( $parameters[ $position ]['raw'] );
+					break;
+			}
+
+			if ( $parameter_args['value'] === $matched_parameter ) {
+				continue;
+			}
+
+			$is_dynamic_parameter = $this->phpcsFile->findNext(
+				array( T_CONSTANT_ENCAPSED_STRING, T_ARRAY, T_FALSE, T_TRUE, T_NULL, T_LNUMBER, T_WHITESPACE ),
```

Comment:
Regardless for the check if the parameter is dynamic the code will fail if a comment in placed with the parameters. Do we want to support this code format?

```php
wp_new_user_notification( '', /* deprecated */ 'deprecated' );
wp_new_user_notification( '', null /* deprecated */ );
```
`$parameters[ $position ]['raw']` returns `/* deprecated */ 'deprecated'`

Like I raised in #823 parameters return everything between the two commas instead of just the parameter.